src/test/unit_tests/test_s3_bucket_policy.js

```diff
@@ -956,7 +956,10 @@ mocha.describe('s3_bucket_policy', function() {
         }));
     });
 
-    mocha.it('should be able to use notPrincipal', async function() {
+    mocha.it('should be able to use notPrincipal2', async function() {
```

Comment:
If it should be fixed for all environments, better to skip the test for both of them, you can use `mocha.it.skip`.

Reply:
ok, added skip for now.